index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello world</title>
```

Comment:
kindly, I suggest you insert the CSS file to your HTML to be able to visualize the configuration

Reply:
Oh, I forget that. It is fixed.